assets/geode_objects.js

```diff
@@ -15,10 +19,10 @@ const geode_objects = {
         tooltip: 'Graph',
         image: 'edged_curve.svg',
     },
-    // HybridSolid3D: {
-    //   tooltip: "HybridSolid3D",
-    //   image: "HybridSolid3D.svg",
-    // },
+    HybridSolid3D: {
```

Comment:
Is RegularGrid 2D and 3D missing?

Reply:
Is the SVG file supposed to be the same for both regular grid 2D & 3D or not ?

Reply:
I would say different.

Reply:
let's use the same for now